packages/ses/src/error/tame-console.js

```diff
@@ -79,7 +79,7 @@ export const tameConsole = (
       // eslint-disable-next-line @endo/no-polymorphic-call
       const stackString = loggedErrorHandler.getStackString(event.error);
       // eslint-disable-next-line @endo/no-polymorphic-call
-      causalConsole.error(stackString);
+      causalConsole.error(`${event.error.message}\n${stackString}`);
```

Comment:
Both the before and after seem wrong to me. Why not pass the error itself to the console?

Reply:
Good point. That was my original thought as well - I should have tried that first.

I was also wondering how this dealt with custom error types. I'll try that out in a unit test.

Reply:
This has been updated! I found that errors with custom names were displayed the same either way as well.